src/OpenLoco/IndustryManager.cpp

```diff
@@ -190,18 +190,238 @@ namespace OpenLoco::IndustryManager
         }
     }
 
+    // 0x00459A05
+    static bool isTooCloseToNearbyIndustries(const Map::Pos2& loc)
+    {
+        for (auto& industry : industries())
+        {
+            const auto dist = Math::Vector::manhattanDistance(loc, Map::Pos2{ industry.x, industry.y });
+            if (dist < 480)
+            {
+                return true;
+            }
+        }
+        return false;
+    }
+
+    // 0x00459A50
+    static bool isOutwithCluster(const Map::Pos2& loc, const uint8_t indObjId)
+    {
+        auto numClusters = 0;
+        for (auto& industry : industries())
+        {
+            if (industry.objectId != indObjId)
+            {
+                continue;
+            }
+            numClusters++;
+            const auto dist = Math::Vector::manhattanDistance(loc, Map::Pos2{ industry.x, industry.y });
+            if (dist < 960)
+            {
+                return false;
+            }
+        }
+        if (numClusters < 3)
+        {
+            return false;
+        }
+        return true;
+    }
+
+    // 0x00469A81
+    static int16_t mountainHeight(const Map::Pos2& loc)
+    {
+        // Works out roughly the height of a mountain of area 11 * 11
+        // (Its just the heighest point - the lowest point)
+        int16_t lowest = std::numeric_limits<int16_t>::max();
+        int16_t highest = 0;
+        Map::TilePosRangeView range{
+            loc - Map::TilePos2{ 5, 5 }, loc + Map::TilePos2{ 5, 5 }
+        };
+        for (auto& tilePos : range)
+        {
+            if (!Map::validCoords(tilePos))
+            {
+                continue;
+            }
+            auto tile = Map::TileManager::get(tilePos);
+            auto* surface = tile.surface();
+            auto height = surface->baseHeight();
+            lowest = std::min(lowest, height);
+            if (surface->slope())
+            {
+                height += 16;
+                if (surface->isSlopeDoubleHeight())
+                {
+                    height += 16;
```

Comment:
```suggestion
                height += Map::kMicroZStep;
                if (surface->isSlopeDoubleHeight())
                {
                    height += Map::kMicroZStep;
```

Reply:
I'm not sure that would be the best constant to use. Yes it is 16 and it is related to microzstep but this is really the height of a slope (which happens to be the height of a microzstep). I think I'll leave for a different refactor.